scripts/lua/modules/ts_5min_dump_utils.lua

```diff
@@ -187,7 +187,48 @@ function ts_dump.host_update_stats_rrds(when, hostname, host, ifstats, verbose)
   ts_utils.append("host:unreachable_flows", {ifid = ifstats.id, host = hostname,
 					   flows_as_client = host["unreachable_flows.as_client"],
 					   flows_as_server = host["unreachable_flows.as_server"]},
-		  when, verbose)
+      when, verbose)
+  
+  interface.select(tostring(ifstats.id))
+   
+  local host_ = interface.getHostInfo(hostname)
+   
+  if(host_ ~= nil) then
+   
+    ts_utils.append("host:tcp_pkts_stats", {ifid = ifstats.id, host = hostname,
+              retransmission_packets = host_["tcp.packets.retransmissions"],
+              out_of_order_packets = host_["tcp.packets.out_of_order"],
+              lost_packets = host_["tcp.packets.lost"]},
+        when, verbose)
+   
+    ts_utils.append("host:dns_pkts_sent", {ifid = ifstats.id, host = hostname,
+              queries_packets = host_["dns"]["sent"]["num_queries"],
+              replies_ok_packets = host_["dns"]["sent"]["num_replies_ok"],
+              replies_error_packets = host_["dns"]["sent"]["num_replies_error"]},
+        when, verbose)
+   
+    ts_utils.append("host:dns_pkts_rcvd",{ifid = ifstats.id, host = hostname,
+              queries_packets = host_["dns"]["rcvd"]["num_queries"],
+              replies_ok_packets = host_["dns"]["rcvd"]["num_replies_ok"],
+              replies_error_packets = host_["dns"]["rcvd"]["num_replies_error"]},
+        when, verbose)
+         
+    ts_utils.append("host:udp_pkts", {ifid = ifstats.id, host = hostname,
+              packets_sent = host_["udp.packets.sent"],
+              packets_rcvd = host_["udp.packets.rcvd"]},
+        when, verbose)
+      
+    local stats = interface.getMacInfo(host_["mac"])
```

Comment:
why are you doing `getMacInfo`? mac addresses are already iterated when calling `callback_utils.foreachDevice` in this very same file....

Reply:
because i can't get arp stats from the table host_

Reply:
you can:
- decide to store arp counters in ip addresses (ip addresses in arp packets are already dissected) - in which case you'll have such counters in the host.
- leave arp counters in the mac addresses and use host mac address at query time to retrieve mac arp counters

calling this `getMacInfo` causes unnecessary extra work to be done...

Reply:
Could you explain better the first point, please?

Reply:
We already dissect ARP packets to get sender and target destination addresses. Those guys are IPv4s so you can decide to count ARPs for IPs.